crates/weaver_resolver/data/multi-registry/custom_registry/custom_registry.yaml

```diff
@@ -28,4 +28,7 @@ groups:
       - ref: auction.name
         requirement_level: recommended
       - ref: error.type
-        requirement_level: required
\ No newline at end of file
+        requirement_level: required
+
+imports:
```

Comment:
slack discussion: we will have refs to signals within registry as well, so maybe we can either:
- have `metric_ref` under `group` - there are some implementation constraints there
- have a different section name like `refs`